static/js/components/JobDetails.jsx

```diff
@@ -190,8 +227,9 @@ var JobDetails = React.createClass({
   },
 
   cronUsage: function() {
+    var msg = "Note: To reduce errors, direct modifications aren't currently supported. Please instead deactivate the old job and add a new.";
```

Comment:
Yeah I think that makes sense, but the way Clever does this is hardcode a refresh rate in clever-cron. 

I think, ideally, it's a service that can be merged with Cron Admin, maybe even with Gearman Admin. But given that 1) this is open source and 2) the refresh rate is hard coded, I'm not sure what's ideal. Maybe a nice solution would be to add a button which allows you to refresh clever-cron, however that might be another PR itself.